src/components/interface/MellowLPTable/MellowLPTable.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from "@mui/system";
+import { AugmentedMellowLpVault } from "@utilities";
+import MellowLPEntry from "./components/MellowLPEntry";
+
+
+export type MellowLPTableProps = {
+    lpVaults: AugmentedMellowLpVault[];
+    onSelectItem: (item: AugmentedMellowLpVault) => void;
+    disabled: boolean;
+}
+
+const MellowLPTable: React.FunctionComponent<MellowLPTableProps> = ({
+    lpVaults,
+    onSelectItem,
+    disabled,
+}: MellowLPTableProps) => {
+    const renderContent = () => {
+        return (<Box sx={{display: "flex", justifyContent: "space-between"}}>
+            <MellowLPEntry lpVault={lpVaults[0]} onSelectItem={() => onSelectItem(lpVaults[0])} disabled={disabled}/>
+        </Box>)
+    }
+
+    return renderContent();
```

Comment:
```suggestion
    return (<Box sx={{display: "flex", justifyContent: "space-between"}}>
            <MellowLPEntry lpVault={lpVaults[0]} onSelectItem={() => onSelectItem(lpVaults[0])} disabled={disabled}/>
        </Box>);
```

No need for this kind of "renderContent" stuff. It just uses memory space on every render.

Reply:
Resolved. https://github.com/Voltz-Protocol/voltz-ui/commit/a44c3c48e2a2660e1fc14f6eeeda646152d2edea